.eslintrc.js

```diff
@@ -26,7 +26,7 @@ module.exports = {
 		'jest/globals': true,
 	},
 	globals: {
-		wpApiSettings: true,
+		wpApiSchema: true,
```

Comment:
Is there any way to avoid using this global?

Reply:
Not at the moment, I believe the need for this will go away once we refactor our code to move away from `withApiData`

Reply:
Right, thanks for making it clear :)